src/app/Fake.Core.Process/ProcessUtils.fs

```diff
@@ -7,14 +7,27 @@ open Fake.IO.FileSystemOperators
 
 [<RequireQualifiedAccess>]
 module ProcessUtils =
+    /// Information about a dotnet tool
+    type DotNetTool =
+      {
+        DotNetCli : string option
+        Tool : string option
```

Comment:
Can you add some docs what this "Tool" string means and what "None" means?

Reply:
Will do in next update, when I've got something working for `withDotNetTool` that I'm happy with.